Dispatcher/Controllers/HomeViewController.swift

```diff
@@ -26,4 +36,66 @@ class HomeViewController: UIViewController {
         
         
     }
+    
+    
+}
+
+
+extension HomeViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return articlesToDisplay.count
+        
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: ArticleCell.articleCellIdentifier, for: indexPath) as! ArticleCell
```

Comment:
Using as! is risky because if for some reason we can't make the forced downcast then our application will crash.
It's better practice to use "as?" and alternatively return a regular UITableCell because we prefer the user to see an empty cell rather than having the application crash.